Jellyfin.Server/Program.cs

```diff
@@ -65,36 +63,9 @@ static Task ErrorParsingArguments(IEnumerable<Error> errors)
                 .MapResult(StartApp, ErrorParsingArguments);
         }
 
-        /// <summary>
-        /// Shuts down the application.
-        /// </summary>
-        internal static void Shutdown()
-        {
-            if (!_tokenSource.IsCancellationRequested)
-            {
-                _tokenSource.Cancel();
-            }
-        }
-
-        /// <summary>
-        /// Restarts the application.
-        /// </summary>
-        internal static void Restart()
-        {
-            _restartOnShutdown = true;
-
-            Shutdown();
-        }
-
         private static async Task StartApp(StartupOptions options)
         {
             _startTimestamp = Stopwatch.GetTimestamp();
-
-            // Log all uncaught exceptions to std error
-            static void UnhandledExceptionToConsole(object sender, UnhandledExceptionEventArgs e) =>
```

Comment:
Is this not needed anymore?

Reply:
The default behavior is already to log to console, so this was logging everything to the console twice.